scripts/ubuntu/openstack.sh

```diff
@@ -0,0 +1,10 @@
+apt-get -y install pump cloud-utils cloud-init cloud-initramfs-growroot \
+    bash-completion
+
+# Ignore wardning about datasources
+mkdir -p /etc/cloud/cloud.cfg.d
+cat <<EOF >/etc/cloud/cloud.cfg.d/99-warnings.cfg;
+#cloud-config
+warnings:
+  dsid_missing_source: off
+EOF
```

Comment:
Just to document it here. I'd recommend instead of turning off this warning to update the config to use the correct datasource. You can do something similar to what we did in the [old bento repo for Debian](https://github.com/osuosl/bento/blob/master/packer/scripts/debian/openstack.sh#L7-L17).